Description: Adds a clock fix. It was improved by Robert Scheck <robert@fedoraproject.org> 
to work with older Linux kernel versions, too.
Author: <kees@ubuntu.com>
Origin: http://pkgs.fedoraproject.org/cgit/rpms/dsniff.git/tree/dsniff-2.4-sysconf_clocks.patch
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/sshow.c
+++ b/sshow.c
@@ -217,7 +217,9 @@
 {
 	clock_t delay;
 	int payload;
-    long CLK_TCK= sysconf(_SC_CLK_TCK);
+#if defined(_SC_CLK_TCK)
+	long CLK_TCK = sysconf(_SC_CLK_TCK);
+#endif
 
 	delay = add_history(session, 0, cipher_size, plain_range);
 
@@ -266,7 +268,9 @@
 	clock_t delay;
 	int skip;
 	range string_range;
-    long CLK_TCK= sysconf(_SC_CLK_TCK);
+#if defined(_SC_CLK_TCK)
+	long CLK_TCK = sysconf(_SC_CLK_TCK);
+#endif
 	
 	delay = add_history(session, 1, cipher_size, plain_range);
 	
